doc/sinter_api.md

```diff
@@ -161,11 +161,11 @@ class CollectionOptions:
             biggest batch size that can finish in under the given number of
             seconds. Limits each batch to be no larger than that.
     """
-    max_shots: Optional[int] = None
```

Comment:
This is a major degradation in the documentation. I'll have to fix it before this can be merged.